renovate.json

```diff
@@ -3,5 +3,9 @@
     "extends": ["config:recommended"],
     "timezone": "America/Los_Angeles",
     "schedule": ["after 12pm on thursday"],
-    "semanticCommitScope": ""
+    "semanticCommitScope": "",
+    "postUpgradeTasks": {
+        "commands": ["yarn build"],
```

Comment:
So this is pretty cool - by adding this, renovate will now run `yarn build` before creating the commit allowing updates like this to include the pre-processed files 🥳